projects/addon-preview/components/package.json

```
@@ -0,0 +1,10 @@
+{
+    "ngPackage": {
+        "lib": {
+            "entryFile": "index.ts",
+            "styleIncludePaths": [
+                "../../../core/styles"
```

i guess it should be:

```suggestion
                "../../core/styles"
```

---

but check it, please)

---

no, with 2 '/' build falls)